x-pack/plugins/monitoring/server/lib/metrics/apm/metrics.js

```diff
@@ -449,4 +449,142 @@ export const metrics = {
       }
     ),
   }),
+  apm_acm_response_count: new ApmEventsRateClusterMetric({
+    field: 'beats_stats.metrics.apm-server.acm.response.count',
+    title: i18n.translate('xpack.monitoring.metrics.apm.acmResponse.countTitle', {
+      defaultMessage: 'Response Count Agent Configuration Management',
+    }),
+    label: i18n.translate('xpack.monitoring.metrics.apm.acmResponse.countLabel', {
+      defaultMessage: 'Count',
+    }),
+    description: i18n.translate('xpack.monitoring.metrics.apm.acmResponse.countDescription', {
+      defaultMessage: 'fill in',
+    }),
+  }),
+  apm_acm_response_errors_count: new ApmEventsRateClusterMetric({
+    field: 'beats_stats.metrics.apm-server.acm.response.errors.count',
+    title: i18n.translate('xpack.monitoring.metrics.apm.acmResponse.errorCountTitle', {
+      defaultMessage: 'Response Error Count Agent Configuration Management',
+    }),
+    label: i18n.translate('xpack.monitoring.metrics.apm.acmResponse.errorCountLabel', {
+      defaultMessage: 'Error Count',
+    }),
+    description: i18n.translate('xpack.monitoring.metrics.apm.acmResponse.errorCountDescription', {
+      defaultMessage: 'fill in',
```

Comment:
TODO: Need copy from APM team

Reply:
@bmorelli25 do you have any good ideas for a description here?

Reply:
I'm not sure if I can be of much help here. Looking at the descriptions in https://github.com/elastic/kibana/pull/43998 here's at least a starting point. The APM Server team will need to elaborate:

**Response Count Agent Configuration Management (/s)**

* Count: _HTTP requests responded to by APM Server_
* Error Count: _HTTP errors count_
* OK: _200 OK response count_
* Not modified: _304 Not modified response count_

**Response Errors Agent Configuration Management (/s)**

* Count: 
* Unauthorized: _Unauthorized HTTP requests rejected count_
* Unavailable: _Unavailable HTTP response count. Possible misconfiguration or unsupported version of Kibana_
* Method: _HTTP requests rejected due to incorrect HTTP method_
* Invalid query: _Invalid HTTP query_

**Requests Agent Configuration Management (/s)**

* Count: _HTTP Requests received by agent configuration management_

Reply:
Updated!